components/HomeHeader.vue

```diff
@@ -179,137 +67,59 @@ export default Vue.extend({
 
       this.relatedItems[key] = itemData;
     }
-    this.loading = false;
+
+    if (this.items.length === 0) {
+      this.extraText = this.$t('homeHeader.welcome.noItems').toString();
+    } else {
+      this.extraText = this.$t('homeHeader.welcome.checkNewItems').toString();
+    }
+    window.setTimeout(this.hideWelcomeMessage, 1500);
   },
   methods: {
-    ...mapActions('playbackManager', ['play']),
-    getRelatedItem(item: BaseItemDto): BaseItemDto {
-      const rItem = this.relatedItems[this.items.indexOf(item)];
-      if (!rItem) {
-        return item;
-      }
-      return rItem;
-    },
-    getOverview(item: BaseItemDto): string {
-      if (item.Overview) {
-        return this.sanitizeHtml(item.Overview);
+    hideWelcomeMessage(): void {
+      if (this.items.length === 0) {
+        const elem = this.$refs.headerWelcome as HTMLElement;
+        // As the height of the element varies as we're using automatic values, we set first
+        // the screen height of the element, so the animation plays correctly when the height is set to 0
+        elem.style.maxHeight = elem.scrollHeight + 'px';
+        elem.classList.add('no-items');
+        elem.addEventListener('animationend', this.onNoItemsTransitionEnd);
       } else {
-        return '';
+        this.loading = false;
       }
     },
-    getLogo(item: BaseItemDto): string {
-      const relatedItem = this.getRelatedItem(item);
-      return this.getImageUrlForElement(ImageType.Logo, {
-        itemId: relatedItem.Id
-      });
-    },
-    // HACK: Vue-awesome-swiper seems to have a bug where the components inside of duplicated slides (when loop is enabled,
-    // swiper creates a duplicate of the first one, so visually it looks like you started all over before repositioning all the DOM)
-    // doesn't get the parameters passed correctly on components that calls to methods. Whenever the beginning or the end is reached,
-    // we force a BlurhashImage reload to fix this by updating it's key.
-    //
-    // TODO: Revisit this once we are using the original Swiper.js library.
-    forceReload(): void {
-      this.reloadSentinel = 1;
-      this.reloadSentinel = 0;
+    onNoItemsTransitionEnd(): void {
+      (this.$refs.headerWelcome as Element).remove();
+      this.$destroy();
```

Comment:
It'd be better to use v-if to do this, as it's essentially the same and is cleaner.